great_expectations/rule_based_profiler/data_assistant_result/data_assistant_result.py

```diff
@@ -194,9 +194,13 @@ def show_expectations_by_expectation_type(
         include_profiler_config: bool = False,
         send_usage_event: bool = True,
     ) -> None:
-        """
-        Populates named "ExpectationSuite" with "ExpectationConfiguration" list, stored in "DataAssistantResult" object,
-        and displays this "ExpectationConfiguration" list, grouped by "expectation_type", in predetermined order.
+        """Populates named "ExpectationSuite" with "ExpectationConfiguration" list stored in "DataAssistantResult"
+        object and displays this "ExpectationConfiguration" list, grouped by "expectation_type", in predetermined order.
```

Comment:
Can we shorten this? And backticks will probably render better than quotes around entity names